libmscore/edit.cpp

```diff
@@ -1013,7 +1013,36 @@ NoteVal Score::noteValForPosition(Position pos, bool &error)
                   }
 
             case StaffGroup::STANDARD: {
-                  AccidentalVal acci = s->measure()->findAccidental(s, staffIdx, line, error);
+                  AccidentalVal acci;
+                  if (! _is.pitchMod())  // if no raise or lower
+                        acci = s->measure()->findAccidental(s, staffIdx, line, error);
+                  else  // raise or lower half step
+                  {
+                        // find what this note would be just from the key signature
+                        // (first ChordRest segment in the measure)
+                        acci = s->measure()->findAccidental(s->measure()->first(Segment::Type::ChordRest), staffIdx, line, error);
+                        // add pitchMod to that
+                        switch (static_cast<int>(acci) + _is.pitchMod())
```

Comment:
As far as the information I can find, (int)acci and int(acci) in C++ are equivalent, and they both have the same possible problems and dangers.

I originally wrote int(acci), but then I saw the MuseScore coding rules and changed it to
static_cast<int>(acci)

Should I change this back?